docs/api-docs/liftState.md

```diff
@@ -0,0 +1,43 @@
+# `liftState(state): [any, Cmd]`
+
+* `state: any` &ndash; an object which may be the state of the redux store, or
+  an existing `[any, Cmd]` pair created by `loop()`.
+
+## Notes
+
+Automatically converts objects to `loop()` results. If the value was created
+with `loop()`, then the function behaves as an identity. Otherwise, it is lifted
+into a `[any, Cmd]` pair where the effect is `Cmd.none()`. Useful for
```

Comment:
can you remove the parentheses on Cmd.none? It's not a function

Reply:
Oh, right! will do.